sql/log_event_server.cc

```diff
@@ -5969,7 +5969,9 @@ int Rows_log_event::do_apply_event(rpl_group_info *rgi)
       slave_rows_error_report(ERROR_LEVEL, thd->is_error() ? 0 : error,
                               rgi, thd, table, get_type_str(),
                               RPL_LOG_NAME, log_pos);
-    if (thd->slave_thread)
+    if (IF_WSREP(thd->slave_thread ||
```

Comment:
Looks a little strange having `thd->slave_thread` twice for both A and B conditions - perhaps just `#ifdef WITH_WSREP` around your conditions would look nicer.